crypto/fipsmodule/ec/p384.c

```diff
@@ -164,19 +160,40 @@ static void p384_felem_cmovznz(p384_limb_t out[P384_NLIMBS],
   }
 }
 
-// NOTE: the input and output are in little-endian representation.
 static void p384_from_generic(p384_felem out, const EC_FELEM *in) {
+#ifdef OPENSSL_BIG_ENDIAN
+  uint8_t tmp[48];
+  bn_words_to_little_endian(tmp, 48, in->words, P384_NLIMBS);
```

Comment:
can we add `#define P384_FELEM_BYTES (48)` and use the macro instead of constants here and in `p384_from_scalar`?